compiler/src/dotty/tools/dotc/core/NullOpsDecorator.scala

```diff
@@ -0,0 +1,144 @@
+package dotty.tools.dotc.core
+
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Symbols.defn
+import dotty.tools.dotc.core.Types.{AndType, ClassInfo, ConstantType, OrType, Type, TypeBounds, TypeMap, TypeProxy}
+
+/** Defines operations on nullable types. */
+object NullOpsDecorator {
```

Comment:
We should integrate `Types.OrNull` here and remove any duplications. Right now we have stripNull twice. I believe we can drop the one in `Types.OrNull`.

Reply:
The `stripNull` in `Types.OrNull` is removed.